NEWS.md

```diff
@@ -1,5 +1,7 @@
 # evaluate (development version)
 
+* The `source` output handler is now parsed the entire top-level expression, not just the first component.
```

Comment:
I was forced to fix this buglet as part of the refactoring. I'll need to make a fix to pkgdown, and again rely on the dev version of evaluate.

Reply:
Oops, I meant "passed" not "parsed".